core/src/main/kotlin/org/evomaster/core/problem/externalservice/httpws/HarvestActualHttpWsResponseHandler.kt

```diff
@@ -60,22 +61,20 @@ class HarvestActualHttpWsResponseHandler {
     @Inject
     private lateinit var randomness: Randomness
 
-    private lateinit var httpWsClient : Client
+    private lateinit var httpWsClient: Client
 
 
     /**
-     * TODO
-     * to further improve the efficiency of collecting actual responses,
-     * might have a pool of threads to send requests
+     * TODO: Add EMConfig option to set value as config
      */
-    private lateinit var threadToHandleRequest: Thread
+    private var workerPool = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors())
```

Comment:
it might be too many if we run tasks in parallel.
how about `min(3, Runtime.getRuntime().avaiableProcessors())` in case a less powerful laptop?

Reply:
`Runtime.getRuntime().avaiableProcessors()` will return number of processor available for the JVM. However, I changed it to as suggested.